include/ginkgo/core/log/stream.hpp

```diff
@@ -188,7 +188,7 @@ class Stream : public Logger {
     static std::unique_ptr<Stream> create(
         std::shared_ptr<const Executor> exec,
         const Logger::mask_type& enabled_events = Logger::all_events_mask,
-        std::ostream& os = std::cout, bool verbose = false)
+        std::ostream& os = std::cerr, bool verbose = false)
```

Comment:
Probably better to have `clog`, so that it is properly buffered and in order ?

Reply:
how would that improve results? using clog means that log messages may be delayed an arbitrary amount of time, and buffering has no impact on the ordering of messages.